frontend/src/components/Modals/RollbackModal.tsx

```diff
@@ -0,0 +1,102 @@
+import {Button, InputNumber, message, Modal, Space} from 'antd';
+import {useEffect, useState} from "react";
+import {HelmRelease} from "../../types";
+import {rollbackRelease} from "../../api";
+import {useNavigate} from "react-router";
+
+const RollbackModal = ({open, setOpen, release}: {
+    open: boolean,
+    setOpen: (open: boolean) => void,
+    release: HelmRelease | undefined
+}) => {
+    const [confirmLoading, setConfirmLoading] = useState(false);
+    const [revision, setRevision] = useState(0);
+    const [messageApi, contextHolder] = message.useMessage();
+
+    const navigate = useNavigate();
+
+    useEffect(() => {
+        const newRevision = parseInt(release?.revision || "0");
+        setRevision(newRevision);
+    }, [release]);
+
+    const success = () => {
+        messageApi.open({
+            type: 'success',
+            content: 'Rollback successful.',
+            duration: 2,
+            key: 'rollback',
+        })
+    }
+
+    const loading = () => {
+        messageApi.open({
+            type: 'loading',
+            content: 'Rollback will continue in the background.',
+            duration: 2,
+            key: 'rollback',
+        })
+    }
```

Comment:
I can't see this messages anywhere

Reply:
```
if ('chart' in result) {
                **success();**
```

Reply:
In app. I asume that these are messages

Reply:
Yeah. Weird. Maybe i changed something. I will look into this.

Reply:
Yeah, I did something pretty stupid before. Now fixed.